pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -643,17 +645,30 @@
     "    headers = {\n",
     "        \"unstructured-api-key\": request.headers.get(\"unstructured-api-key\")\n",
     "    }\n",
+    "    \n",
+    "    # Retry parameters\n",
+    "    retry_enabled = bool(os.environ.get(\"UNSTRUCTURED_PARALLEL_RETRY_ENABLED\", False))\n",
```

Comment:
I'd say default should probably be true.

Also you'll need to explicitly check `var == "true"`, otherwise casting a string to a bool will run into this:
```
In [2]: bool("false")
Out[2]: True
```

Reply:
thanks, i think if we want default to be true, can i just remove this env?

Reply:
Yeah, that makes sense. If we want to disable we can set retries to 0.